src/server/json_family_test.cc

```diff
@@ -50,7 +50,7 @@ TEST_F(JsonFamilyTest, SetGetBasic) {
     </store>
 )";
 
-  auto resp = Run({"set", "json", json});
+  auto resp = Run({"SET", "json", json});
```

Comment:
why did you change the case? it should work regardless.

Reply:
I tried to replace a specific word but I clicked on 'replace all' by mistake.
I'm going to revert this change